packages/react/src/components/calendar/Calendar.tsx

```diff
@@ -33,14 +33,55 @@ interface I18nConfig {
 
 export interface ICalendarOwnProps extends React.ClassAttributes<Calendar> {
     id?: string;
+    /**
+     * The list of months available in the calendar
+     *
+     * @default ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']
+     */
     months?: string[];
+    /**
+     * The month initially displayed in the calendar in the form of an index of the "months" array prop
+     *
+     * @default current month
+     */
     startingMonth?: number;
+    /**
+     * The list of years available in the calendar
```

Comment:
```suggestion
     * The array of years available in the calendar
```

because you use "array" instead of list in other props descriptions ^^

Reply:
I actually use `list` in the other props description 🤔 . I only use array when talking about the index of something